### ISSUE TITLE
Track Progress

### ISSUE THREAD

**Issue by slgraham** [CommitPool/CommitPool-monorepo#40]
**User Story / Job to be Done / Problem**
- As a user with an active commitment, I want to know how much remaining distance until I reach my goal so that I know much more activity I need to do.
- As a user with an active commitment, I want to know how much more time I have in my commitment period so I can plan when to get that remaining distance.
- As a user with an active commitment, I want to know how much DAI I will lose if I don't complete my goal on time in order to motivate myself to get that distance.
- As a user with an active commitment, I want to double check my current activity in my Strava profile so that I can be sure my info is up to date.

**Potential solution ideas**
- mvp: link to strava profile
- more advanced: embed strava workouts / distances within commitment period
- 

**Additional context**

**Comment by bitbeckers** [comment]
Screen is there, first need to connect to contract for commitment and strava for activity data

**Comment by slgraham** [comment]
I think mainly this is a UX/UI task. I'll play around with some things in the figma

**Comment by bitbeckers** [comment]
Yeah, but I'll make sure the active commitment is available in state
